instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/client.py

```diff
@@ -102,6 +114,22 @@ def _finish_tracing_callback(future, span, response_hook):
                 description=description,
             )
         )
+    response = future.result()
```

Comment:
You can use `response` when getting the status code above as well.

Reply:
Fixed